pace-util/HISTORY.md

```diff
@@ -7,13 +7,16 @@ latest
 Major changes:
 - Renamed DummyComm to LocalComm, and added support for message tags. The DummyComm symbol is still in place for backwards compatibility, but points to LocalComm
 - added error in CubedSphereCommunicator init if given a communicator with a size not equal to the total ranks of the given partitioner
+- `subtile_extent` method of Partitioner classes now takes in a required `rank` argument
+- TilePartitioner has a new `edge_interior_ratio` argument which defaults to 1.0, and lets the user specify the relative 1-dimensional extent of edge and corner compute domains compared to interior compute domains. In all cases, the closest valid value will be used, which enables some previously invalid configurations (e.g. C128 on a 3 by 3 layout will use the closest valid edge_interior_ratio to 1.0)
```

Comment:
```suggestion
- TilePartitioner has a new `edge_interior_ratio` argument which defaults to 1.0, and lets the user specify the relative 1-dimensional extent of the boundaries (edge and corner) compute domains compared to interior compute domains. In all cases, the closest valid value will be used, which enables some previously invalid configurations (e.g. C128 on a 3 by 3 layout will use the closest valid edge_interior_ratio to 1.0)
```

Trying to align the wording with the code (e,g, `ratio between interior and boundary tile sizes.` in function comment). Boundary is a bit of a loaded term here, so you might want to flip for a constructed "edge-corner" concept or something better.

Reply:
Will do.

Reply:
Done.